.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Scala CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        java: ['adopt@1.8', 'adopt@1.11']
```

Comment:
We don't want to publish for both Java versions

Reply:
should i just default to 1.11 here?

Reply:
Yeah I think it's fine to drop Java 8 already